integrations/aws/main.py

```diff
@@ -0,0 +1,168 @@
+import json
+from typing import Any
+
+import boto3
+from port_ocean.core.models import Entity
+from utils import ACCOUNT_ID_PROPERTY, ResourceKindsWithSpecialHandling, describe_resources, describe_single_resource, _fix_unserializable_date_properties, _get_sessions, find_account_id_by_session, get_resource_kinds_from_config, update_available_access_credentials, validate_request, get_matching_kinds_from_config
+from port_ocean.context.ocean import ocean
+from loguru import logger
+from starlette.requests import Request
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+
+@ocean.on_resync()
+async def resync_all(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    if kind in iter(ResourceKindsWithSpecialHandling):
+        logger.info("Kind already has a specific handling, skipping", kind=kind)
+        return
+    async for batch in resync_cloudcontrol(kind):
+        yield batch
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.CLOUDRESOURCE)
+async def resync_generic_cloud_resource(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    resource_kinds = get_resource_kinds_from_config(kind)
+
+    if len(resource_kinds) == 0:
+        logger.error(f"Resource kinds not found in port app config for {kind}")
+        return
+
+    for kind in resource_kinds:
+        async for batch in resync_cloudcontrol(kind):
+            yield batch
+
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.ACM)
+async def resync_acm(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    sessions = _get_sessions()
+    for batch in describe_resources(sessions, 'acm', 'list_certificates', 'CertificateSummaryList', 'NextToken'):
+        yield batch
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.LOADBALANCER)
+async def resync_loadbalancer(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    sessions = _get_sessions()
+    for batch in describe_resources(sessions, 'elbv2', 'describe_load_balancers', 'LoadBalancers', 'NextMarker'):
+        yield batch
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.CLOUDFORMATION)
+async def resync_cloudformation(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    sessions = _get_sessions()
+    for batch in describe_resources(sessions, 'cloudformation', 'list_stacks', 'StackSummaries', 'NextToken'):
+        yield batch
+
+
+async def resync_cloudcontrol(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    sessions = _get_sessions()
```

Comment:
I thought you already have the describe_resources function to do it, why you need this function as well? also it seems like it doesn't handle resources such as `LOADBALANCER` etc..

Reply:
Yes the `describe_resources` function is very similar to `resync_cloudcontrol`, I chose to duplicate a lot of the code instead of making `describe_resources` more generic and have her contain `resync_cloudcontrol` functionality, since making `describe_resources` more generic will make her more complex and difficult to maintain and since the majority of our entities will be generate via cloudcontrol I thought it would be easier to debug and maintain separately, WDYT?